src/Microsoft.Health.Fhir.Shared.Core/Extensions/ReindexJobRecordExtensions.cs

```diff
@@ -55,6 +55,7 @@ public static ResourceElement ToParametersResourceElement(this ReindexJobWrapper
             parametersResource.Parameter.Add(new Parameters.ParameterComponent() { Name = JobRecordProperties.MaximumConcurrency, Value = new FhirDecimal(job.MaximumConcurrency) });
             parametersResource.Parameter.Add(new Parameters.ParameterComponent() { Name = JobRecordProperties.Resources, Value = new FhirString(job.ResourceList) });
             parametersResource.Parameter.Add(new Parameters.ParameterComponent() { Name = JobRecordProperties.SearchParams, Value = new FhirString(job.SearchParamList) });
+            parametersResource.Add(JobRecordProperties.TargetResourceTypes, new FhirString(job.TargetResourceTypeList));
```

Comment:
>parametersResource.Add [](http://example.com/codeflow?start=12&length=22)

It really stands out...
Maybe it's worth to rewrite other parametes as well?

Reply:
Yes, it does look cleaner.